images/linux/scripts/installers/graalvm.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -e
+
+#Install GraalVM
+tmp_graalvm="/tmp/graalvm"
+mkdir -p "$tmp_graalvm"
+url=$(curl -s https://api.github.com/repos/graalvm/graalvm-ce-builds/releases/latest | jq -r '.assets[].browser_download_url | select(contains("graalvm-ce-java11-linux-amd64"))')
+download_with_retries "$url" "$tmp_graalvm/graalvm-archive.tar.gz"
+mkdir /usr/local/graalvm
+tar -xzf "$tmp_graalvm/graalvm-archive.tar.gz" -C /usr/local/graalvm
+setEtcEnvironmentVariable "GRAALVM_BIN" /usr/local/graalvm/graal*/bin
+
+#Install Native Image
+$GRAALVM_BIN/gu install native-image
```

Comment:
I don't think that this variable will be available here. You should set it with `export`

Reply:
Fixed it